### ISSUE TITLE
java.lang.IllegalArgumentException: Layer 61 not used.

### ISSUE THREAD

**Issue by ghost** [TerraFirmaCraft/TerraFirmaCraft#1828]
Server version: 1.12.2
MOD Version: 1.7.21.179
Error message: https://paste.helpch.at/wowupecobi.sql

**Comment by alcatrazEscapee** [comment]
```
at org.bukkit.craftbukkit.v1_12_R1.generator.NormalChunkGenerator.func_185932_a(NormalChunkGenerator.java:41)
```

makes me suspicious, can you reproduce without craftbukkit? If no, then you're out of luck.

**Comment by alcatrazEscapee** [comment]
This issue does not apply, or has been fixed in the current MC 1.18 versions of TFC, and as MC 1.12 is receiving no more development, it will be closed. If this issue arises again in 1.18, please file a new issue report.
